README.md

```diff
@@ -1 +1,14 @@
-# awesome-list
\ No newline at end of file
+# Awesome Projects [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
+
+> A curated list of open source projects from students at KPI
```

Comment:
Why is this using a blockquote? It doesn't look like a citation semantically...

Reply:
Just generated part of file, can replace with anything else. What will be more suitable mark?

Reply:
Let's keep it for now. Markdown is pretty limited...

Or alternatively, have it just wrapped with `<center>`.

Reply:
Ok, let's keep